src/pages/ReferencePage.vue

```diff
@@ -1,162 +1,276 @@
 <template>
   <div>
-    <b-container
-      fluid
-      class="p-4 bg-light"
-    >
-      <b-row>
-        <div class="col-12">
-          <h1> Unsupervised Learning </h1>
-          <p> 
+    <div> 
+      <h2> References </h2>
+      <p>Click on the labels to find out more.</p>
+    </div>
+    <div>
+      <b-button
+        v-b-toggle.collapse-1
+        variant="primary"
+        class="collapsebtnpri"
+      >
+        Unsupervised Learning
+      </b-button>
+      <b-collapse
+        id="collapse-1"
+        class="mt-2"
+      >
+        <b-card>
+          <p class="card-text">
             In this project, we sought to find distinct clusters of start-ups with similar 
             characteristics to uncover insights on their growth prospect. We used K-prototype, 
             K-means, and Hierarchical Clustering to cluster the start-ups. More details about 
             each clustering approach can be found below.
           </p>
-        </div>
-      </b-row>
-      <hr>
+          <b-button
+            v-b-toggle.collapse-1-inner-1
+            size="sm"
+            class="collapsebtnsec"
+          >
+            Results
+          </b-button>
+          <b-collapse
+            id="collapse-1-inner-1"
+            class="mt-2"
+          >
+            <b-card>
+              <div class="col-12">
+                <p>
+                  We compared the results obtained from running the different unsupervised learning methods 
+                  and did not find visually distinct groups of start-ups. The clusters seem to only be distinctly
+                  separable based on total funding amount. 
+                </p>
+              </div>
+              <hr>
+              <img
```

Comment:
the img is no longer centered now it is wrapped in the new tags, are you able to find a workaround for this?